Assets/Locales/Mapper_en.asset

```diff
@@ -675,6 +675,46 @@ MonoBehaviour:
     m_Localized: Chroma Step Gradient
     m_Metadata:
       m_Items: []
+  - m_Id: 144
+    m_Localized: Note
+    m_Metadata:
+      m_Items: []
+  - m_Id: 145
+    m_Localized: Bomb
+    m_Metadata:
+      m_Items: []
+  - m_Id: 146
+    m_Localized: Wall
+    m_Metadata:
+      m_Items: []
+  - m_Id: 147
+    m_Localized: Delete
+    m_Metadata:
+      m_Items: []
+  - m_Id: 148
+    m_Localized: Fade
+    m_Metadata:
+      m_Items: []
+  - m_Id: 149
```

Comment:
Nitpick: Perhaps apply the English localization text to the other languages so people wouldn't be met with errors similar to `Missing localization for key "bar.events.on"`